src/wp-includes/class-wp-customize-widgets.php

```diff
@@ -420,6 +420,7 @@ public function customize_register() {
 				'priority'                 => 110,
 				'active_callback'          => array( $this, 'is_panel_active' ),
 				'auto_expand_sole_section' => true,
+				'theme_supports'           => 'widgets',
```

Comment:
This hides the _Widgets_ panel if the theme doesn't support `'widgets'`. It isn't necessary as it will be hidden anyway because there won't be any registered sidebars. But I thought it would be nice to explicitly add this for consistency with the _Menus_ panel.

Reply:
I'd recommend against this if it's not needed. Avoids unneded code for either 5.9 or 5.9.1.

Reply:
Consistency makes sense here. The Core commit should reference `menus` and `widgets` per the PR's description, making code archaeology easier.

Reply:
Yeah, cool. I was on the fence. I've removed the change.